src/main/java/io/deephaven/csv/CsvSpecs.java

```diff
@@ -170,6 +205,24 @@ public static Builder builder() {
         return ImmutableCsvSpecs.builder();
     }
 
+    /**
+     * Validates the {@link CsvSpecs}.
+     */
+    @Check
+    protected void check() {
```

Comment:
I don't think this should have the protected modifier (wider than it needs to be); should be no modifier.

Reply:
Changed. However, be aware I was copying the style I saw at https://immutables.github.io/immutable.html#precondition-check-method , which does use "protected"